cleanup/artifactCleanup/artifactCleanup.groovy

```diff
@@ -45,7 +45,7 @@ class Global {
 
 def pluginGroup = 'cleaners'
 def config = new ConfigSlurper().parse(new File(ctx.artifactoryHome.haAwareEtcDir, PROPERTIES_FILE_PATH).toURL())
-def regex = ~/-\d+\.\d+\.\d+/
+def regex = ~/.*-[\.\d+]*[-+][\.\d+]*\..*/
```

Comment:
Could you elaborate on this change?

Reply:
Yes. I've made this, because in the past we had a problem with missing artefacts with + instead on - as a delimiter. This pattern is already used in config file for navkit-release-local.